user-guide/monitoring.md

```diff
@@ -225,6 +225,19 @@ spec:
     component: core
 ```
 
+Now, create a service and `Mapping` to expose Grafana behind Ambassador:
+
+```yaml
+---
+apiVersion: getambassador.io/v1
+kind: Mapping
+metadata: 
+  name: grafana
+spec:     
+  prefix: /grafana/
+  service: grafana
```

Comment:
service: grafana:{{GRAFANA_NAMESPACE}} 

The nice thing about specifying the namespace is we can avoid issues with Ambassador's (annoying) behavior of only checking things in it's own namespace.

Reply:
I think this will not be a problem once @kflynn [changes this behaviour](https://datawire.slite.com/app/channels/4AdeaD~eWg/collections/ImTrhhDZKR/notes/5Im92Ri~dk), right?

Reply:
Yes but this is being merged into master so it is currently needed